frontend/src/App.css

```diff
@@ -1,5 +1,10 @@
 .App {
   text-align: center;
+  min-height: 100vh;
+  align-items: center;
+  justify-content: center;
+  background-image: linear-gradient(370deg, orange 50%, cyan);
```

Comment:
why we can a bg image for the `.App` class?
This will apply to all FE components. Please add these styles to a new class and use that class name for your component stylings.